---
layout: page
title: About the PI
---


<img style="float: right;" src="../public/rupingsun.jpg">

Ruping Sun <br>
Assistant Professor <br>
Advanced Cancer Genomics and Bioinformatics <br>
[Department of Laboratory Medicine and Pathology](https://med.umn.edu/bio/lab-med-and-pathology-faculty/ruping-sun) <br>
Member of [Masonic Comprehensive Cancer Center](https://www.cancer.umn.edu/) <br>
University of Minnesota Twin Cities

[@Sun_Ruping](http://twitter.com/Sun_Ruping) ---
[Google Scholar](https://scholar.google.de/citations?user=bRpxn-oAAAAJ&hl=en) 

ruping at umn dot edu *

As a computational cancer biologist and geneticist, my research focuses on computational innovations to detect somatic genomic variants from massive parallel sequencing data, and mathematical approaches to identify functional events by leveraging the evolutionary dynamics of cancer. During my research tenure, I have developed considerable expertise in the analysis of genomic variants and mathematical modeling of cancer evolution. My expertise in cancer genomics originated from my first postdoc training with Prof. Martin Vingron in Germany, where I implemented several algorithms for detecting sequence breakpoints from DNA and RNA sequencing data. To explore the roles of somatic variants in tumor development, I first investigated how somatic variants propagate in gene regulatory networks during my additional postdoctoral training with Prof. Andrea Califano at Columbia University. There I realized that variants at different scales are intertwined with each other (e.g., point mutations and somatic copy number aberrations) and their roles cannot be fully understood without an evolutionary perspective. Being a research staff at Stanford University, I focused on characterizing cancer clonal evolution through the analysis of multi-region, longitudinal and single-cell sequencing datasets of clinical samples. My rationale is that one can quantify intra tumor heterogeneity (ITH) from patient tumor sequencing data and in turn leverage the patterns of ITH to infer the mode (or the way) of clonal evolution in vivo (Sun, Nat. Genet. 2017). This approach will delineate the roles of the genomic aberrations on the development of individual patient tumors.

Leading a research group at the University of Minnesota, I continue to bridge ITH with mathematical modeling and interpretability. We prototyped a framework to infer the timing of SCNAs during the somatic evolution toward the founder of clinical samples (Wang et al., bioRxiv, 2022). In addition, we obtained results expressing the genomic divergence between metastatic and primary tumors in terms of the time and mode of metastatic seeding (Sun and Nikolakopoulos, PLoS Comput Biol 2021). My long term goal is to create broadly applicable computational methods that advance a mechanistic understanding of patient tumor evolution and prioritize cancer genes for early detection or as drug targets.

