---
layout: page
title: About the PI
---


<img style="float: right;" src="../public/rupingsun.jpg">

Ruping Sun <br>
Assistant Professor <br>
Advanced Cancer Genomics and Bioinformatics <br>
[Department of Laboratory Medicine and Pathology](https://med.umn.edu/bio/lab-med-and-pathology-faculty/ruping-sun) <br>
Member of [Masonic Comprehensive Cancer Center](https://www.cancer.umn.edu/) <br>
University of Minnesota Twin Cities

[@Sun_Ruping](http://twitter.com/Sun_Ruping) ---
[Google Scholar](https://scholar.google.de/citations?user=bRpxn-oAAAAJ&hl=en) 

ruping at umn dot edu *

With a background in computational biology and cancer genetics, my research centers on cancer genomics and tumor evolution. I am particularly interested in testing fundamental hypotheses about how aneuploidy arises in cancer and how it affects cellular fitness—especially in the context of the recurrent aneuploidy patterns frequently observed in advanced-stage tumors. While model systems provide essential mechanistic insights, my work places a strong emphasis on patient-derived data to capture the complexity of aneuploidy in its native tumor context.

To this end, I have developed substantial expertise in analyzing genomic variants and modeling cancer evolution using patient tumor data. Early in my postdoctoral training, I implemented algorithms to detect structural variants from whole-genome and RNA sequencing of cancer specimens (Sun et al., *Bioinformatics* 2012; Fernandez-Cuesta and Sun et al., *Genome Biol.* 2015). I then shifted focus to characterizing clonal evolution in cancer through the analysis of multi-region, longitudinal, and single-cell sequencing datasets from clinical samples. This led me to incorporate evolutionary dynamics into models of intra-tumor heterogeneity (Sun et al., *Nat. Genet.* 2017; Sun et al., *PLoS Comput. Biol.* 2021) and to estimate clinically relevant parameters for patient tumor growth (Hu et al., *Nat. Genet.* 2019; Leder et al., *J. Math. Biol.* 2024).

My long-term goal is to bridge intra-tumor heterogeneity with analytical modeling to prioritize specific aneuploidies for early detection or therapeutic targeting. By analyzing multi-sampled tumor specimens, I develop computational tools to reconstruct the evolutionary trajectories of aneuploidies during patient tumor development. For example, my group published a framework to infer the timing of aneuploidy acquisition during somatic evolution leading to the tumor founder clone (Wang et al., *Nat. Commun.* 2024). In parallel, we collaborate with experts in cancer aneuploidy to both cross-validate our findings in patient tumors and explore cancer type-specific vulnerabilities linked to recurrent aneuploidy patterns. In collaboration with Dr. Sheltzer, for instance, we showed that tumor cells can become “addicted” to specific aneuploidies, such as trisomy 1q (Girish et al., *Science* 2023). With these experiences, I am well-prepared to lead a team to explore patient data and engage in cross-disciplinary collaborations.

<!--
As a computational cancer biologist, my research focuses on computational innovations to detect somatic genomic variants from massive parallel sequencing data, and mathematical approaches to identify oncogenic events by leveraging the evolutionary dynamics of cancer. My rationale is that one can quantify intra tumor heterogeneity (ITH) from patient tumor sequencing data (such as multi-region, longitudinal and single-cell sampling data) and in turn leverage the patterns of ITH to infer the mode (or the way) of clonal evolution in vivo. This approach will delineate the roles of the genomic aberrations on the development of individual patient tumors. Leading a research group at the University of Minnesota, I am driven to bridge ITH with mathematical modeling and interpretability. We prototyped a framework to infer the timing of SCNAs during the somatic evolution toward the founder of clinical samples (Wang et al., bioRxiv, 2022). In addition, we obtained results expressing the genomic divergence between metastatic and primary tumors in terms of the time and mode of metastatic seeding (Sun and Nikolakopoulos, PLoS Comput Biol 2021). My long term goal is to create broadly applicable computational methods that advance a mechanistic understanding of patient tumor evolution and identify cancer genes for early detection or as drug targets.
-->