---
layout: post
title: Theory of Tumor Heterogeneity
---

We innovate computational algorithms to quantify and model tumor heterogeneity (TH) from next generation sequencing (NGS) data, and leverage patterns of TH to statistically infer the kinetics of cancer.

<div style="float:left; border:solid 1px gray; margin:10px;"><img src="../public/clonal_dynamics.png" width="230"></div>

---
**_Tumor Clonal Dynamics_**: we perform `data driven modeling` of single-cell based spatial tumor growth, to reveal the principles that govern the relation between genomic heterogeneity, clonal dynamics, and sequencing detectability of somatic variants.

---

[//]: # <div style="float:none;clear:both;"></div>
[//]: # ![cd](../public/clonal_dynamics.png# thumbnail bordered){: style="float:left; padding:1px"}


<div style="float:left; border:solid 1px gray; margin:10px;"><img src="../public/analytics_TH.png" width="230"></div>

---
**_Computational Genomics in Tumor Heterogeneity_**: We push the limits in computational methods to achieve a high-fidelity quantification of (epi)genomic heterogeneity from tumor sequencing data (including CNA, SV, SNV, Gene Expression etc). Biases are inescapable but can be informative when we understand them better.
<!-- Example: [VAP](https://combine-lab.github.io/VAP/) -->

---

<div style="float:left; border:solid 1px gray; margin:10px;"><img src="../public/ITH_plot.png" width="230"></div>

---
**_Visualizing Genomic Heterogeneity_**: Data visualizaion plays critical role in computational genomics. We develop visualization tools for tumor sequencing data to reveal the patterns of tumor heterogeneity, facilitate feature extraction, identify potential biases and inspire hypotheses regarding the dynamics of human cancers.
<!-- Example: [VAP](https://combine-lab.github.io/VAP/) -->

---
